Assets/GothicVR/Scripts/Creator/WorldCreator.cs

```diff
@@ -25,7 +25,7 @@ public static class WorldCreator
         private static GameObject _worldGo;
         private static GameObject _teleportGo;
         private static GameObject _nonTeleportGo;
-        private static HashSet<IPolygon> _claimedPolygons;
+        private static readonly HashSet<IPolygon> ClaimedPolygons;
```

Comment:
ClaimedPolygons is never initialized and so when trying to check if the hashset contains polygon, we get an error
Please Fix